### ISSUE TITLE
Броня у СБ

### ISSUE THREAD

**Issue by RandomColonist** [animusdev/blue#621]
На этот раз серьёзно. Мадман, ты сделал хуйню. Даже у зимней куртки капитана броня лучше, чем у  жилетки и шлема обычного щитсека. Она не защитит даже от мелкашки, не то, что от чего покрупнее. Подкрути показатели брони хотя бы немного, а так же впили броню детективу и криминалисту в их плащи/куртки обратно. Желательно - дать хороший бафф броне вардена и хоса, ибо они не лучше говнистых жилеток СБ, а ещё лучше - дать им при этом два кармашка, ибо почему нет?

**Comment by Ahmedok** [comment]
Сейчас бронежилет кэпа лучше, чем все стандартные броники есбе. А в арсенале у нас лежит боевая броня, которая по показателям может посоревноваться с дедсквадовской, лол. Я не знаю, кто добавлял в арсенал боевую и тактическую брони, но он явно любит поногебать за есбе (мадман).

**Comment by RandomColonist** [comment]
Броня полностью отсосная, даже с такими показателями. Дело в том, что она пиздец медленная, а шлем в боевом режиме работает как сварочная маска, бля.

**Comment by RandomColonist** [comment]
Нет, бля, серьёзно, у нынешней практически ВСЕЙ брони сб самые нищенские показатели, буквально не выше 20-30, максимум - "barely defends". Я в куртке вардена, например, сосу от первой же пульки в грудь, в то время как на бее она с переменным успехом может впитать несколько ударов и пару выстрелов. Серьёзно, поднимите показатели хотя-бы на 10-20 очков и будет самое то.
